### DISCUSSION TITLE
Cannot make a Copilot subscription

### DISCUSSION THREAD

**Discussion by TOM-12** [community/community#59680] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Want to confirm my payment information, but it shows this screen
![image](https://github.com/community/community/assets/5844053/4c643181-ab40-4830-8376-d8dd215ffa2f)
Whatever I do cannot progress
tried different browser, private mode, disabled all adblocks and such, still same...

**Comment by chris97pl** [comment]
Various GitHub systems are down at the moment. Keep an eye on https://www.githubstatus.com/
